### PR TITLE
Implement Estimated Earnings indicator 2.0

### PR DESCRIPTION
**Pull request by sujitacharya2005**
Resolves https://github.com/brave/brave-browser/issues/30352

## Submitter Checklist:

- [x] I confirm that no security/privacy review [is needed](https://github.com/brave/brave-browser/wiki/Security-reviews), or that I have [requested](https://github.com/brave/security/issues/new/choose) one
- [x] There is a [ticket](https://github.com/brave/brave-browser/issues) for my issue
- [x] Used Github [auto-closing keywords](https://docs.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue) in the PR description above
- [x] Wrote a good [PR/commit description](https://google.github.io/eng-practices/review/developer/cl-descriptions.html)
- [x] Squashed any review feedback or "fixup" commits before merge, so that history is a record of what happened in the repo, not your PR
- [x] Added appropriate labels (`QA/Yes` or `QA/No`; `release-notes/include` or `release-notes/exclude`; `OS/...`) to the associated issue
- [x] Checked the PR locally:
  * `npm run test -- brave_browser_tests`, `npm run test -- brave_unit_tests` [wiki](https://github.com/brave/brave-browser/wiki/Tests)
  * `npm run lint`, `npm run presubmit` [wiki](https://github.com/brave/brave-browser/wiki/Presubmit-checks), `npm run gn_check`, `npm run tslint`
- [x] Ran `git rebase master` (if needed)

## Reviewer Checklist:

- [ ] A security review [is not needed](https://github.com/brave/brave-browser/wiki/Security-reviews), or a link to one is included in the PR description
- [ ] New files have MPL-2.0 license header
- [ ] Adequate test coverage exists to prevent regressions
- [ ] Major classes, functions and non-trivial code blocks are well-commented
- [ ] Changes in component dependencies are properly reflected in `gn`
- [ ] Code follows the [style guide](https://chromium.googlesource.com/chromium/src/+/HEAD/styleguide/c++/c++.md)
- [ ] Test plan is specified in PR before merging

## After-merge Checklist:

- [ ] The associated issue milestone is set to the smallest version that the
  changes has landed on
- [ ] All relevant documentation has been updated, for instance:
  - [ ] https://github.com/brave/brave-browser/wiki/Deviations-from-Chromium-(features-we-disable-or-remove)
  - [ ] https://github.com/brave/brave-browser/wiki/Proxy-redirected-URLs
  - [ ] https://github.com/brave/brave-browser/wiki/Fingerprinting-Protections
  - [ ] https://github.com/brave/brave-browser/wiki/Brave%E2%80%99s-Use-of-Referral-Codes
  - [ ] https://github.com/brave/brave-browser/wiki/Custom-Headers
  - [ ] https://github.com/brave/brave-browser/wiki/Web-Compatibility-Exceptions-in-Brave
  - [ ] https://github.com/brave/brave-browser/wiki/QA-Guide
  - [ ] https://github.com/brave/brave-browser/wiki/P3A

## Test Plan:
Changes
1. replacement poppins font to san-serrif
2. tooltip handling from onTochInterCept
               [ View estimatedTooltipGroup = mPopupView.findViewById(R.id.estimated_tooltip_group);](https://github.com/brave/brave-core/blob/d5b9996336759b6888c13e1421a7ab1081de5753/android/java/org/chromium/chrome/browser/rewards/BraveRewardsPanel.java#L263)

3. estimated range
4. if there is no range show 0.000
5. show progress bar with loading text
6.  for grand parent state it will show "- - Learn More"

| image    |   Screenshot_20230529_161819    |  image    |  image    |       image   |    Screenshot_20230529_134450  |   image   |    image   |
|-|-|-|-|-|-|-|-|

### CODE REVIEW

File: android/java/org/chromium/chrome/browser/BraveRewardsNativeWorker.java [hunk 1]
```diff
@@ -708,10 +708,11 @@ public void onGetAvailableCountries(String[] countries) {
 
     @CalledByNative
     public void OnGetAdsAccountStatement(boolean success, double next_payment_date,
-            int ads_received_this_month, double earnings_this_month, double earnings_last_month) {
+            int ads_received_this_month, double min_earnings_this_month,
+            double max_earnings_this_month, double earnings_last_month) {
         for (BraveRewardsObserver observer : mObservers) {
             observer.OnGetAdsAccountStatement(success, next_payment_date, ads_received_this_month,
-                    earnings_this_month, earnings_last_month);
+                    min_earnings_this_month, max_earnings_this_month, earnings_last_month);
```
Comment (deeppandya) [inline android/java/org/chromium/chrome/browser/BraveRewardsNativeWorker.java:715]: nit : java method param should be in camelcase

Comment (sujitacharya2005) [reply android/java/org/chromium/chrome/browser/BraveRewardsNativeWorker.java]: updated

File: browser/ui/android/strings/android_brave_strings.grd [hunk 2]
```diff
@@ -774,6 +780,7 @@ This file contains all "about" strings.  It is set to NOT be translated, in tran
       <message name="IDS_P3A_INSIGHTS" desc="Text for p3a insights onboarding">
         Share completely private and anonymous product insights about what features are being used by Brave's users.
       </message>
+
```
Comment (SergeyZhukovsky) [inline browser/ui/android/strings/android_brave_strings.grd:783]: why that empty line has been added?

**Review by SergeyZhukovsky — APPROVED** [review]
++

**Review by deeppandya — APPROVED** [review]
LGTM

**Review by mkarolin — APPROVED** [review]
`strings`++

**Pull request merged**